### ISSUE TITLE
SLuggish performance when opening play store

### ISSUE THREAD

**Issue by videeshhh** [sebanc/brunch#1755]
i have 12thgen i5 12400 6 core processor with 16gigs of ram ive installed chromeos in triple boot with brya recovery and brunch r107 both are same version but when i click on play store it load like hell took 5 mins to load everything works fine but slow on startup and it is installed in ssd pls help me to fix this issue

**Comment by flobo09** [comment]
Indeed, it looks like an issue when updating to the latest chrome OS 109.0.5414.9. 

The whole android subsystem is so slow it's almost unusable.
